modules/lib/lib-project/src/main/resources/lib/xp/examples/project/create.js

```diff
@@ -0,0 +1,96 @@
+// BEGIN
+// Creates a Content Project with minimal properties
+var projectLib = require('/lib/xp/project');
+try {
+    var project = projectLib.create({
+        id: 'my-project',
+        displayName: 'My Content Project',
+        readAccess: {
+            public: true
+        }
+    });
+} catch (e) {
+    log.error('Failed to create a project: ' + e);
+}
+// END
+
+// BEGIN
+var expected = {
+    'id': 'my-project',
+    'displayName': 'My Content Project',
+    'permissions': [],
+    'readAccess': {
+        'public': true
+    }
+};
+// END
+
+// BEGIN
+// Creates a Content Project with extended properties
+var projectLib = require('/lib/xp/project');
+var project = projectLib.create({
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['project' is already defined.](https://app.codacy.com/manual/enonic/xp/pullRequest?prid=5511138)